src/control/common/storage/commands.go

```diff
@@ -23,6 +23,7 @@ type StoragePrepareNvmeCmd struct {
 	PCIBlockList string `long:"pci-block-list" description:"Whitespace separated list of PCI devices (by address) to be ignored when unbinding devices from Kernel driver to be used with SPDK (default is no PCI devices)."`
 	NrHugepages  int    `short:"p" long:"hugepages" description:"Number of hugepages to allocate for use by SPDK (default 1024)"`
 	TargetUser   string `short:"u" long:"target-user" description:"User that will own hugepage mountpoint directory and vfio groups."`
+	DisableVFIO  bool   `long:"disable-vfio" description:"Force SPDK to use the UIO driver for NVMe device access"`
```

Comment:
yes there should be a doc update, this would be a user request though and probably not something we want to autodetect, I've added it to be consistent with daos_server working in daemon mode so we have the same behaviour. I will update the admin guide. (it's an option to override use of VFIO even if enabled)

Reply:
will be addressed by follow-on stand-alone doc update PR